frontend/www/js/omegaup/components/Overlay.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <div class="overlay" v-if="showOverlay" data-overlay>
+      <slot name="popup-content"></slot>
+    </div>
+  </div>
```

Comment:
```suggestion
  <div v-if="showOverlay" data-overlay>
    <slot name="popup-content"></slot>
  </div>
```

no necesitamos el `<div>` de afuera ni la clase! pero lo que sí se necesita es que si se hace click en el `data-overlay`, se oculte el overlay.

tons hay necesidad de que se cree una propiedad con el valor inicial de `showOverlay`:

```typescript
 @Prop({ default: false }) showOverlay!: boolean;
isOverlayShown = showOverlay;
```

probablemente haya que tener un `@Watcher` también.

Reply:
A ver si está bien como lo implementé